object-manipulation-3/main.js

```diff
@@ -0,0 +1,125 @@
+console.log('Lodash is loaded:', typeof _ !== 'undefined');
+
+var players = createPlayers(4);
+playGame(players, 2);
+
+function playGame(playerList, numCards) {
+  var deck = createDeck();
+  deck = _.shuffle(deck);
+  for (var i = 0; i < playerList.length; i++) {
+    playerList[i].hand = drawCards(deck, numCards);
+  }
+  console.log('Player\'s Hands:', playerList);
+  getScores(playerList);
+  var winner = getWinner(playerList);
+  var newPlayerList = checkForTies(playerList, winner);
+  if (newPlayerList.length > 1) {
+    console.log('A tie has occurred so a tie-breaker game will run');
+    playGame(newPlayerList, 2);
+  } else {
+    console.log('The winner is', winner.name);
```

Comment:
You should put a space after `is` in your string so that the message is formatted correctly